test/epictest/three_legged_cat_test.go

```diff
@@ -71,7 +71,7 @@ func RunThreeLeggedCat(data []byte, conf testutil.LatencyConfig) error {
 		return err
 	}
 
-	readerCatted, err := coreunix.Cat(catter, keyAdded)
+	readerCatted, err := coreunix.Cat(catter, keyAdded.String())
```

Comment:
May be worth to create a `Path` interface that Key can implement, and that we can make a small wrapper for strings, to let us do:

```go
coreunix.Cat(catter, key)
coreunix.Cat(catter. ipfs.Path("/foobar")
```

What I'm worried about is this:

```go
coreunix.Cat(catter, key.Pretty())
coreunix.Cat(catter, key.String())
coreunix.Cat(catter, string(key))
```
Which one of these is right? they cant all be.